jdaviz/configs/default/plugins/export/export.py

```diff
@@ -207,6 +221,20 @@ def _set_subset_not_supported_msg(self, msg=None):
         else:  # no subset selected (can be '' instead of None if previous selection made)
             self.subset_invalid_msg = ''
 
+    def _set_dataset_not_supported_msg(self, msg=None):
+        if self.dataset.selected_obj is not None:
+            data_unit = self.dataset.selected_obj.unit
+            if data_unit == u.Unit('DN/s'):
+                self.data_invalid_msg = f'Export Disabled: The unit {data_unit} could not be saved in native FITS format.'  # noqa: E501
+            if isinstance(self.dataset.selected_obj, NDData):
+                self.data_invalid_msg = "Export is not implemented for NDdata objects."

```

Comment:
if Spectrum1D inherits from NDData, then this won't work.  My suggestion would be to mirror the same logic used when actually exporting.